<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="MSI" failures="1" tests="1" name="Test2(failed)" time="5.247" errors="0" timestamp="2023-08-10T10:52:53 IST">
  <testcase classname="com.testNG.TestNGParallel2" name="launchLambdatest" time="1.525">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testNG.TestNGParallel2.launchLambdatest(TestNGParallel2.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- launchLambdatest -->
  <testcase classname="com.testNG.TestNGParallel2" name="launchSaucelabs" time="0.0">
    <ignored/>
  </testcase> <!-- launchSaucelabs -->
</testsuite> <!-- Test2(failed) -->
